docs/userguide.md

```diff
@@ -93,6 +93,9 @@ automated, then invoke the deployment script by running:
 ./scripts/deploy-federation-latest.sh cluster2
 ```
 
+The above script will join the host cluster to federation control panel automatically.
+In above case, it means join both cluster1 and cluster2 to the federation control panel.
```

Comment:
Can be modified to:
The argument used is the context name of the additional clusters that needs to be joined to this federation control plane.
Clarifying, say the `host-cluster-context` used is `cluster1`, then on successful completion of the script used in example, both `cluster1` and `cluster2` will be joined to the deployed federation control plane.

Reply:
Done